### PR TITLE
Remove functionality related to the scan static report

### PR DESCRIPTION
**Pull request by manelmontilla**
This PR removes the functionality related to the static scan report that we are going to deprecate.
Concretely this PR removes:
* All the endpoints related to the static report:
GET /api/v1/teams/{team_id}/scans/{scan_id}/report
POST /api/v1/teams/{team_id}/scans/{scan_id}/report
POST /api/v1/teams/{team_id}/scans/{scan_id}/report/send
GET /api/v1/teams/{team_id}/scans/{scan_id}/report/email
* The functionality in the `vulcan-reports-generator` client related to the scan reports
That functionality is also being removed from the [vulcan-report-generator itself ](https://github.com/adevinta/vulcan-reports-generator/pull/117)
* Remove the logic that was listening and processing the events for finished scans as it's not needed anymore.
* Remove the commands from the `vulcan-cli` that were dealing with static scan reports.

### CODE REVIEW

**Review by jesusfcr — CHANGES_REQUESTED** [review]
Very impressive!
Added a couple of comments.

File: contrib/dev/local-dev.toml [hunk 1]
```diff
@@ -30,14 +30,6 @@ url = "http://localhost:8082/"
...
-enabled = false
-log_messages = true
-polling_interval = 10
-wait_time = 20
-timeout = 30
-queue_arn = "arn:aws:sqs:xxx:123456789012:yyy"
-
 [reports]
 sns_arn = "arn:aws:sns:xxx:123456789012:yyy"
 api_base_url = "http://localhost:8084"
```
[the remark this answers is not in the document]
Comment (manelmontilla) [reply contrib/dev/local-dev.toml]: Good catch thank you, removing them.

File: pkg/api/endpoint/reports.go [hunk 2]
```diff
@@ -14,80 +14,6 @@ import (
...
-		err = s.GenerateReport(ctx, reportRequest.TeamID, team.Name, reportRequest.ScanID, false)
-		if err != nil {
-			_ = logger.Log("ErrGenerateReport", err)
-		}
-
-		return Accepted{}, nil
-	}
-}
-
-func makeFindReportEndpoint(s api.VulcanitoService, logger kitlog.Logger) endpoint.Endpoint {
```
[the remark this answers is not in the document]
Comment (manelmontilla) [reply pkg/api/endpoint/reports.go:44]: Absolutely good catch, thank you.

**Review by jesusfcr — APPROVED** [review]
LGTM

**Pull request merged**